src/components/current-score.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const CurrentScore = (props) => (
+    <div className="player-current-box">
+        <div className="player-current-label">Current</div>
```

Comment:
Good practice is to implement internationalize at the start of a project - react-intl